db/migrate/20140818155533_add_index_to_feature_decisions.rb

```diff
@@ -0,0 +1,7 @@
+require 'activerecord-concurrent-index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.